charts/gsp-cluster/values.yaml

```diff
@@ -389,3 +389,19 @@ flux:
   namespace: flux-system
   helmOperatorRole: ""
 
+external-dns:
+  sources:
+    - ingress
+    - istio-gateway
+  provider: aws
+  aws:
+    region: "eu-west-2"
+  rbac:
+    create: true
+    apiVersion: v1
+
+istioSystem:
+  permittedRolesRegex: ""
+
+letsencrypt:
+  email: daniel.blair@digital.cabinet-office.gov.uk
```

Comment:
you happy with your email being here? I'm assuming you'll get a million Let's Encrypt emails because of it.

Reply:
It'll be removed in the next card. Agile FTW!